content/en/docs/getting-started.md

```diff
@@ -4,26 +4,30 @@ weight: 1
 excerpt: ""
 ---
 
+Rebar3 is the standard build tool within the Erlang community. It essentially bundles many of the other tools shipping with Erlang along with a few open-source ones, and makes them all work under a unified project structure.
```

Comment:
"essentially bundles" -- it doesn't bundle the OTP tools, maybe some better wording for how it bundles the use of those tools but not the tools themselves?

Reply:
Yeah. Funnily enough I reused the AdoptingErlang blurb, but I can just swap it for "wraps" or "integrates"